src/core/style/mixins.scss

```diff
@@ -114,6 +114,7 @@
 @mixin rtl-prop($ltr-prop, $rtl-prop, $value) {
   #{$ltr-prop}: $value;
   [dir=rtl] & {
+    #{$ltr-prop}: auto;
```

Comment:
You're right, i thought that when a property is getting a wrong value it behaves as the initial behavior, silly me!